### ISSUE TITLE
No Error hanlder

### ISSUE THREAD

**Issue by andrzejWilde** [asafdav/ng-clip#66]
Hey, there's no error handler which makes using the lib quite difficult.

**Comment by levithomason** [comment]
Agreed, just spent a fair amount time debugging because the path the to the swf was wrong:

    ngClipProvider.setPath('/bower_components/zeroclipboard/dist/ZeroClipboard.swf');

In this case, the bower install location changed from `/app/bower/` to the usual location.  Everything but this module worked.  I'd be willing to help with some PRs if @asafdav is still planning on maintaining this module.

The only indication I had was that my cpu began to peg and the fans turned on :P

**Comment by asafdav** [comment]
Hi, I'm am planning to maintain this lib, what kind of PR would you like to
propose ?

**Comment by levithomason** [comment]
Currently, when something goes wrong there is no indication of where to look or what went wrong.

Perhaps one place to start could be bundling the dependency so the user doesn't have to point to the SWF. Another option might be throwing an error if the SWF is not found at the location specified.
